5001-6000/LIT5549TribesApostles.xml

```diff
@@ -65,17 +68,33 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <ptr target="bm:Budge1899ContendingsText"/>
                         <citedRange>49–50</citedRange>
                     </bibl>
+                    <bibl>
+                        <ptr target="bm:Perruchon1897Notes"/>
+                        <citedRange unit="page">403–404</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Grebaut1913Apotres"/>
+                        <citedRange unit="page">206–207</citedRange>
+                    </bibl>
                 </listBibl>
                 <listBibl type="translation">
                     <bibl>
                         <ptr target="bm:Budge1901ContendingsTransl"/>
                         <citedRange>40-41</citedRange>
                     </bibl>
+                    <bibl>
+                        <ptr target="bm:Perruchon1897Notes"/>
+                        <citedRange unit="page">404–405</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Grebaut1913Apotres"/>
+                        <citedRange unit="page">207–208</citedRange>
+                    </bibl>
                 </listBibl>
                 <listRelation>
                     <relation name="saws:formsPartOf" active="LIT5549TribesApostles" passive="LIT1461Gadlah"/>
                 </listRelation>
```

Comment:
add
 `<relation name="betmas:formerlyAlsoListedAs" active="LIT5549TribesApostles" passive="LIT5849GenealogyApostles"></relation>`
if you are sure that these are doublets and the second is removed

Reply:
ok